sov-modules/sov-modules-api/src/lib.rs

```diff
@@ -40,7 +41,17 @@ impl AsRef<[u8]> for Address {
 impl AddressTrait for Address {}
 
 /// Default implementation of AddressTrait for the module system
-#[derive(borsh::BorshDeserialize, borsh::BorshSerialize, Debug, PartialEq, Clone, Eq)]
+// TODO: decide if we want to feature gate the serde implementations
```

Comment:
lets add issue number to all this TODOS: https://github.com/Sovereign-Labs/sovereign/issues/175

Reply:
Done!